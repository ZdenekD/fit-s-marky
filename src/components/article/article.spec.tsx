import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Article from '.';

describe('Article', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Article>Article</Article>);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-article');

        expect(component.exists()).toBe(true);
    });
});
