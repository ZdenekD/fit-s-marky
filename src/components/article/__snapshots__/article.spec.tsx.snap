// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article match snapshot 1`] = `
<article class="article "
         data-test="component-article"
>
  Article
</article>
`;
